lib-ee/emqx_ee_bridge/test/emqx_ee_bridge_redis_SUITE.erl

```diff
@@ -0,0 +1,412 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2022 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+-module(emqx_ee_bridge_redis_SUITE).
+
+-compile(nowarn_export_all).
+-compile(export_all).
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("common_test/include/ct.hrl").
+-include_lib("snabbkaffe/include/snabbkaffe.hrl").
+
+-include_lib("emqx_bridge/include/emqx_bridge.hrl").
+
+%%------------------------------------------------------------------------------
+%% CT boilerplate
+%%------------------------------------------------------------------------------
+
+-define(REDIS_TOXYPROXY_CONNECT_CONFIG, #{
+    <<"server">> => <<"toxiproxy:6379">>
+}).
+
+-define(COMMON_REDIS_OPTS, #{
+    <<"password">> => <<"public">>,
+    <<"command_template">> => [<<"RPUSH">>, <<"MSGS">>, <<"${payload}">>],
+    <<"local_topic">> => <<"local_topic/#">>
+}).
+
+-define(BATCH_SIZE, 5).
+
+-define(PROXY_HOST, "toxiproxy").
+-define(PROXY_PORT, "8474").
+
+all() -> emqx_common_test_helpers:all(?MODULE).
```

Comment:
Nit: I wonder if we could repeat the tests in a matrix-like fashion for each of the 3 Redis types using groups, so it's easier to select only a single type for quicker test iteration, and when something fails it's easier to debug which config caused it?

Reply:
First I tried to organize the tests with groups 😅 like in influx tests, but things started to seem a bit too implicit for me, so I made an attempt to use a more explicit approach

Reply:
✅